examples/blob-maker/README.md

```diff
@@ -1,6 +1,14 @@
-# Blob Maker App
+# Blob maker app
 
-## How to start locally (Drag and drop functionality will not work locally see below):
+**&nbsp;ℹ&nbsp;Note**:
+
+- Drag and drop functionality doesn't work if you serve the app on localhost. \
+  To make drag and drop available, deploy the app to a publicly accessible location.
```

Comment:
@marcospinello Is this true? I don't think it is, I don't have issues with it locally when I try

Reply:
Oh, I see that this was in here before, but was moved from beneath. I think this is actually true, but specific to this app example. I will see if I can clarify that to not confuse devs that drag/drop doesn't work locally